simplified-books-controller/src/main/java/org/nypl/simplified/books/controller/BookBorrowTask.kt

```diff
@@ -712,18 +711,10 @@ class BookBorrowTask(
         }
       })
 
-    return if (wantFulfill) {
+    if (wantFulfill) {
       this.runAcquisitionFulfill(feedEntry.feedEntry)
     } else {
-      val exception = IllegalStateException()
-      val message =
-        this.services.borrowStrings.borrowBookBorrowAvailabilityInappropriate(availability)
-      this.steps.currentStepFailed(
```

Comment:
At this point everything has worked as expected. The step hasn't failed and we should return `StepSucceeded` instead.